evm/src/test/resources/org/hyperledger/besu/evm/precompile/pointEvaluationPrecompile.json

```diff
@@ -11,11 +11,6 @@
       "FailureMode": "busted versioned hash",
       "Valid": false
     },
-    {
```

Comment:
Why did this test case get removed?

Reply:
Because it is now handled in the constructor of VersionedHash and isn't a failure case for the precompile itself. Corresponding test for the VersionedHash class itself is pending.